master/internal/rm/kubernetesrm/spec.go

```diff
@@ -129,6 +132,103 @@ func (p *pod) configureEnvVars(
 	return envVars, nil
 }
 
+func (p *pod) configureProxyResources() []gatewayProxyResource {
+	if p.exposeProxyConfig == nil {
+		return nil
+	}
+
+	var resources []gatewayProxyResource
+	// TODO(RM-275/gateways) think about experiments, should they proxy every pod, or only rank 0?
+	for i, proxyPort := range p.req.ProxyPorts {
```

Comment:
are these the right ports to use? 

I think our proxy ports stuff is kinda confusing so I would pay close attention here